packages/terra-draw-google-maps-adapter/src/terra-draw-google-maps-adapter.spec.ts

```diff
@@ -695,17 +675,13 @@ describe("TerraDrawGoogleMapsAdapter", () => {
 				map,
 			});
 
-			const mockQuerySelector = jest.spyOn(document, "querySelector");
-
+			const mockQuerySelector = jest.spyOn(container, "querySelector");
 			mockQuerySelector.mockImplementationOnce(
 				() => ({ classList: { add: jest.fn() } }) as unknown as Element,
 			);
 
 			adapter.setCursor("pointer");
 
-			// Once in constructor, once in setCursor
-			expect(map.getDiv).toHaveBeenCalledTimes(2);
```

Comment:
How often that function was called should be considered an implementation detail, or am I missing something here?

Reply:
I think you're probably right, the number of calls to the div is not really relevant from a desired outcome perspective